apps/spruce/src/constants/requesters.ts

```diff
@@ -0,0 +1,43 @@
+import { StringMap } from "types/utils";
+
+const adhocRequester = "ad_hoc";
+const commitQueueRequester = "merge_test";
+const githubMergeRequester = "github_merge_request";
+const githubPRRequester = "github_pull_request";
+const gitTagRequester = "git_tag_request";
+const gitterRequester = "gitter_request";
+const patchRequester = "patch_request";
+const triggerRequester = "trigger_request";
+
+const requesterToTitle: StringMap = {
+  [githubPRRequester]: "GitHub Pull Request",
+  [patchRequester]: "Patch Request",
+  [gitTagRequester]: "GitHub Tag Request",
+  [gitterRequester]: "Gitter Request",
+  [triggerRequester]: "Trigger Request",
+  [adhocRequester]: "Periodic Build",
+  [githubMergeRequester]: "GitHub Merge Request",
+};
+
+const requesterToDescription: StringMap = {
```

Comment:
Do you think these are all necessary? I think a lot of them feel redundant; only `gitterRequester` and `triggerRequester` seem to provide new-ish information

Reply:
were you thinking something like this would be better?:
<img width="431" alt="Screenshot 2024-07-23 at 7 19 59 PM" src="https://github.com/user-attachments/assets/ad55489d-0e38-4d15-8583-7708367e86b1">

Reply:
Yes! 💯

Reply:
updated